filebeat/input/filestream/input_integration_test.go

```diff
@@ -736,3 +737,174 @@ func TestFilestreamTruncateBlockedOutput(t *testing.T) {
 	cancelInput()
 	env.waitUntilInputStops()
 }
+
+// test_symlinks_enabled from test_harvester.py
+func TestFilestreamSymlinksEnabled(t *testing.T) {
+	env := newInputTestingEnvironment(t)
+
+	testlogName := "test.log"
+	symlinkName := "test.log.symlink"
+	inp := env.mustCreateInput(map[string]interface{}{
+		"paths": []string{
+			env.abspath(testlogName),
+			env.abspath(symlinkName),
```

Comment:
The test might not capture everything going on here. The configuration looks like the original log file is collected and that we collect via symlinks. Is this on purpose? If so, what is the purpose of this test?

Reply:
I copied the configuration from `paths` from the original test incorrectly. I have also made the same mistake in the other 3 tests. I have fixed the paths and the corresponding assertions.